LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2017, 
+Copyright (c) 2017,
```

Comment:
@lassoan If you'd rather not have this trimming of whitespace for the License file I can remove. I just had trim whitespace for the entire repo because it was adding unrelated diffs to the deprecated API updates I was doing in vtkSlicerMarkupsToModelLogic.cxx.